online-workflow/src/main/java/eu/itesla_project/online/db/OnlineDbMVStore.java

```diff
@@ -250,7 +252,7 @@ private void storeStepMetadata(String workflowId, String stateId, OnlineStep ste
 
     private Map<String, String> getMetrics(String workflowId, String mapName) {
         if (isWorkflowStored(workflowId)) {
-            HashMap<String, String> metrics = new HashMap<String, String>();
+            TreeMap<String, String> metrics = new TreeMap<>();
```

Comment:
I wanted to stress the fact that the structure is an ordered map. I find it more readable, this way ...